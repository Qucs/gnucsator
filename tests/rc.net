# Qucs 0.0.19

R:R1 _net0 _net1 R="50 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
VProbe:Pr1 _net1 gnd
.TR:TR1 Type="lin" Start="0" Stop="10m" Points="11" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
Vrect:V1 _net0 gnd U="1 V" TH="1 ms" TL="1 ms" Tr="1 ns" Tf="1 ns" Td="0 ns"
C:C1 _net1 gnd C="1 pF" V=""
