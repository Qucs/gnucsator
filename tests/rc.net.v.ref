--- rc.net.v.out	2018-09-19 02:01:30.967945871 +0100
+++ rc.net.v.ref	2018-09-19 01:49:18.763975897 +0100
@@ -1,4 +1,5 @@
 
+vsource #(.dc(0.)) p (.p(gnd),.n(0));
 R #(.R(50),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85),.ic(NA( NA))) R1 (.p(_net0),.n(_net1));
 Vprobe #() Pr1 (.p(_net1),.n(0));
 Vrect #(.U(1),.Td(0n),.Tr(1n),.Tf(1n),.TH(1m),.TL(1m)) V1 (.p(_net0),.n(0));
