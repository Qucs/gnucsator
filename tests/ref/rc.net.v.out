
module main ();
//# Qucs 0.0.19 
R #(.R(50),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85)) R1 (.p(_net0),.n(_net1));
VProbe #() Pr1 (.p(_net1),.n(0));
Vrect #(.TH(1m),.TL(1m),.Td(0n),.Tf(1n),.Tr(1n),.U(1)) V1 (.1(_net0),.2(0));
C #(.C(1u)) C1 (.p(_net1),.n(0));
endmodule // main

